tests/python/pants_test/backend/python/tasks/test_python_distribution_integration.py

```diff
@@ -179,25 +178,26 @@ def test_pants_resolves_local_dists_for_current_platform_only(self):
 
   def _get_current_platform_string(self):
     return Platform.create().resolve_platform_specific({
+      # TODO: this will fail on everyone's laptop if they're not at 10.12.
```

Comment:
Do you have a more actionable TODO? What's the fix?

How about use 'current' as suggested below.

Reply:
Using `'current'` allows us to remove this method (in 4456105)!